src/components/Molecules/ContentsLive.tsx

```diff
@@ -11,9 +11,10 @@ const LiveContents = () => {
   const [update, setUpdate] = useState(false)
   const [isMouseHover, setIsMouseHover] = useState(false)
   const articles = () => {
-    if (articlesData.slice(0, 3).length === 3) {
-      return articlesData.slice(0, 2)
-    }
+    if (articlesData)
```

Comment:
```suggestion
    if (articlesData)
```
これってなんのために追加した感じですか？

Reply:
アドバイスありがとうございます！
それは間違って追加してしまった感じです！
すみません💦